references.bib

```diff
@@ -1,9 +1,119 @@
+  title={Scaling agile software development through lean governance},
+  author={Ambler, Scott W},
+  booktitle={Proceedings of the 2009 ICSE workshop on software development governance},
+  pages={1--2},
+  year={2009},
+  organization={IEEE Computer Society}
+}
+
+@inproceedings{laanti2008implementing,
+  title={Implementing program model with agile principles in a large software development organization},
+  author={Laanti, Maarit},
+  booktitle={Computer Software and Applications, 2008. COMPSAC'08. 32nd Annual IEEE International},
+  pages={1383--1391},
+  year={2008},
+  organization={IEEE}
+}
+
+@article{stoica2013software,
+  title={Software development: Agile vs. traditional},
+  author={Stoica, Marian and Mircea, Marinela and Ghilic-Micu, Bogdan},
+  journal={Informatica Economica},
+  volume={17},
+  number={4},
+  pages={64},
+  year={2013},
+  publisher={INFOREC Association}
+}
+
+@inproceedings{petrillo2010agility,
+  title={Is agility out there?: agile practices in game development},
+  author={Petrillo, Fabio and Pimenta, Marcelo},
+  booktitle={Proceedings of the 28th ACM International Conference on Design of Communication},
+  pages={9--15},
+  year={2010},
+  organization={ACM}
+}
+
+@inproceedings{politowski2016old,
+  title={Are the Old Days Gone? A Survey on Actual Software Engineering Processes in Video Game Industry},
+  author={Politowski, Cristiano and Fontoura, Lisandra and Petrillo, Fabio and Gu{\'e}h{\'e}neuc, Yann-Ga{\"e}l},
+  booktitle={Games and Software Engineering (GAS), 2016 IEEE/ACM 5th International Workshop on},
+  pages={22--28},
+  year={2016},
+  organization={IEEE}
+}
+
+@inproceedings{bick2016inter,
+  title={Inter-Team Coordination in Large Agile Software Development Settings: Five Ways of Practicing Agile at Scale},
+  author={Bick, Saskia and Scheerer, Alexander and Spohrer, Kai},
+  booktitle={Proceedings of the Scientific Workshop Proceedings of XP2016},
+  pages={4},
+  year={2016},
+  organization={ACM}
+}
+
+@article{maruping2009control,
+  title={A control theory perspective on agile methodology use and changing user requirements},
+  author={Maruping, Likoebe M and Venkatesh, Viswanath and Agarwal, Ritu},
+  journal={Information Systems Research},
+  volume={20},
+  number={3},
+  pages={377--399},
+  year={2009},
+  publisher={INFORMS}
+}
+
+
+
+
+@article{ambler2009agile,
+  title={The agile scaling model (ASM): adapting agile methods for complex environments},
+  author={Ambler, Scott W},
+  journal={Environments},
+  year={2009}
+}
+
+@article{ambler2009applying,
+  title={Applying agile and lean principles to the governance of software and systems development},
+  author={Ambler, SW and Kroll, P},
+  journal={IBM Software Rational},
+  year={2009}
+}
+
+
+
+
+@misc{AgileManifesto,
+  title = {{Agile Manifesto} Principles of Agile},
+  howpublished = {\url{http://agilemanifesto.org/iso/en/manifesto.html}},
+  author = {Kent Beck, Mike Beedle, Arie van Bennekum}
+}
```

Comment:
When did you access this? Please see the example: https://gist.github.com/Adrir/0cc01803ef326857b71b6926f8f8bdcd